bgstally/fleetcarrier.py

```diff
@@ -331,15 +376,20 @@ def _get_items(self, category: FleetCarrierItemType = None) -> tuple[list|None,
             case FleetCarrierItemType.COMMODITIES_SELLING:
                 # Lookup name and display name are the same for commodities as we are not passed localised name from CAPI. We
                 # convert the display name later
-                return self.commodities_selling, 'name', 'name', 'stock'
+                return self.commodities_selling, 'name', 'locName', 'stock'
             case FleetCarrierItemType.COMMODITIES_BUYING:
                 # Lookup name and display name are the same for commodities as we are not passed localised name from CAPI. We
                 # convert the display name later
-                return self.commodities_buying, 'name', 'name', 'outstanding'
+                return self.commodities_buying, 'name', 'locName', 'outstanding'
+            case FleetCarrierItemType.CARGO:
+                # Return cargo items
+                return get_by_path(self.data, ['cargo'], []), 'commodity', 'locName', 'qty'
```

Comment:
Should this not simply be:

```python
return get_by_path(self.cargo, 'commodity', 'locName', 'qty'
```

To follow the same pattern?  `self.cargo` should be initialised by this point.